dae/dae/import_tools/tests/test_custom_region_length.py

```diff
@@ -0,0 +1,253 @@
+from copy import deepcopy
+import pytest
+import os
+from os.path import join
+from dae.configuration.gpf_config_parser import GPFConfigParser
+from dae.import_tools import import_tools
+import pyarrow.parquet as pq
+
+
+def test_import_task_bin_size(gpf_instance_2019, tmpdir):
```

Comment:
this test is huge... consider splitting it into multiple tests... or at least add some comments